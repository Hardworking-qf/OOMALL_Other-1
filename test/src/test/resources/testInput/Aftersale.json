{
  "createAftersale01": {
    "type": 0,
    "quantity": 1,
    "reason": "for test",
    "regionId": 0,
    "detail": "this is detailed address",
    "consignee": "testuser",
    "mobile": "12300010001"
  }
}