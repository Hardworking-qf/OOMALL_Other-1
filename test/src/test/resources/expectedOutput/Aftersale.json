{
  "getAftersaleAllStates01": {
    "errno": 0,
    "data": [
      {
        "code": 0,
        "name": "待管理员审核"
      },
      {
        "code": 1,
        "name": "待买家发货"
      },
      {
        "code": 2,
        "name": "买家已发货"
      },
      {
        "code": 3,
        "name": "待店家退款"
      },
      {
        "code": 4,
        "name": "待店家发货"
      },
      {
        "code": 5,
        "name": "店家已发货"
      },
      {
        "code": 6,
        "name": "审核不通过"
      },
      {
        "code": 7,
        "name": "取消"
      },
      {
        "code": 8,
        "name": "已结束"
      }
    ],
    "errmsg": "成功"
  }
}